mtags/shared/src/main/scala-3/scala/scala/meta/internal/pc/SymbolPrinter.scala

```diff
@@ -0,0 +1,36 @@
+package scala.meta.internal.pc
+
+import dotty.tools.dotc.printing.PlainPrinter
+import dotty.tools.dotc.core.Contexts.Context
+import dotty.tools.dotc.core.Symbols._
+import dotty.tools.dotc.core.Names._
+import dotty.tools.dotc.core.Types._
+import dotty.tools.dotc.core.Flags
+import scala.language.implicitConversions
+
+class SymbolPrinter(implicit ctx: Context) extends PlainPrinter(ctx) {
```

Comment:
This uses PlainPrinter from Dotty, which doesn't have the needed method exposed. There is a lot of utility classes that me might end up copying or at least extending from.